librz/analysis/meta.c

```diff
@@ -339,7 +339,7 @@ RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64
 			break;
 		case 0:
 		case 1:
-		case '*':
+		case RZ_OUTPUT_MODE_RIZIN:
```

Comment:
no need to define this if it goes into the default switch case.

Reply:
Should I change it back to `'*'` or should I remove the three cases altogether?